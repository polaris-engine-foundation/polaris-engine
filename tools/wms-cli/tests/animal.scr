func main() {
    print_animal("cat", 4, 0);
    print_animal("dog", 4, 0);
    print_animal("human", 2, 0);
    print_animal("bird", 2, 1);
}

func print_animal(name, legs, wings) {
    print("a " + name + " has " + legs + " legs.");
    if(wings) {
        print("Oh, a " + name + " has wings!");
    }
}
