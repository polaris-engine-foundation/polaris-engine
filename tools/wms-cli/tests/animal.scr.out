a cat has 4 legs.
a dog has 4 legs.
a human has 2 legs.
a bird has 2 legs.
Oh, a bird has wings!
